src/layers/tile/infinite.rs

```diff
@@ -9,6 +9,7 @@ use crate::{
 
 use super::util::parse_data_line;
 
+/// The raw data of a [`InfiniteTileLayer`]. Does not include a reference to its parent [`Map`](crate::Map).
```

Comment:
Does it make sense to say something about the parent here? It sounds more suitable as documentation for `TileLayerData` to me.